alerts/openshift-virtualization-operator/VMStorageClassWarning.md

```diff
@@ -53,14 +69,7 @@ provisioner: openshift-storage.rbd.csi.ceph.com
 # ...
 ```
 
-The `krbd:rxbounce` option creates a bounce buffer to receive data. The default
-behavior is for the destination buffer to receive data directly. A bounce buffer
-is required if the stability of the destination buffer cannot be guaranteed.
-
-See [Optimizing ODF PersistentVolumes for Windows VMs](https://access.redhat.com/articles/6978371)
-for details.
-
-If you cannot resolve the issue, log in to the
-[Customer Portal](https://access.redhat.com) and open a support case,
-attaching the artifacts gathered during the diagnosis procedure.
+If you cannot resolve the issue, see the following resources:
 
+- [OKD Help](https://www.okd.io/help/)
+- [#virtualization Slack channel](https://kubernetes.slack.com/channels/virtualization)
```

Comment:
This should not be changed. An update in the upstream runbook to have this section here is needed.